index/index.go

```diff
@@ -447,12 +447,12 @@ func (w *Writer) WriteLabelIndex(names []string, values []string) error {
 	return errors.Wrap(err, "write label index")
 }
 
-// writeOffsetTable writes a sequence of readable hash entries.
-func (w *Writer) writeOffsetTable(entries []hashEntry) error {
```

Comment:
ohhhhh this would have been alloc'ing the slice header even though the slice values are pass by reference, nice find

Reply:
Yup